browser/net/url_context.cc

```diff
@@ -23,7 +24,7 @@ void BraveRequestInfo::FillCTXFromRequest(const net::URLRequest* request,
     std::shared_ptr<brave::BraveRequestInfo> ctx) {
   ctx->request_identifier = request->identifier();
   ctx->request_url = request->url();
-  ctx->tab_origin = request->site_for_cookies().GetOrigin();
+  ctx->tab_origin = brave::GetURLOrPDFURL(request->site_for_cookies()).GetOrigin();
```

Comment:
is `ctx->tab_origin` used for anything security-related? if so it probably shouldn't be set to the pdfjs pseudo-origin.

Reply:
no only our internal filtering, but I'll update this to scope it to only adblock+TP or just exclude TP.